Frontend/ReactAndGlobalCSS/README.md

```diff
@@ -0,0 +1,28 @@
+**Title:** Working with Global CSS in ReactJS and ReactNative
+**Creator:** Tenji Tembo
+
+### Summary
+
+ReactJS is one of the most commonly used Front end frameworks in the modern web. It's component-first philosophy drives the direction and consistency of the platform, include areas such as CSS. So much so that CSS is "componentized" into ReactJS files and left unable to take advantage of more global elements found in more custom css libraries, such as heavily modified Bootstrap Libraries. In this talk, I'll talk about the available cohesion of using globally available CSS libraries, while still giving overall control to the component referenced via state management.
+
+I'll break down the following:
+- Global CSS Libraries
+  - CSS setup in traditional monolith setups (Rails, LAMP, Django, etc.)
+  - What do they do well
+    - centralized location
+    - use of preprocessors (SASS/LESS) for better management
+  - What they do poorly
+    - less freedom in more one-off CSS setups due to existing CSS definitions
+    - new CSS can override existing CSS and cause visual errors or behavior
+    - just like the monolith they reside in, size can greatly increase
```

Comment:
They might not reside in a monolith and still have problems. You can have a Frontend Monolith even if your apps are microservices.